### PR TITLE
timeverter: Added timeverter tool

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Added PKGBUILD file for timeverter tool (source: https://github.com/D3vil0p3r/timeverter)

### CODE REVIEW

File: packages/timeverter/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  cd $pkgname
+  mkdir -p "$pkgdir/usr/share/timeverter/src"
+  mkdir -p "$pkgdir/usr/bin"
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/" timeverter.py
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/src/" src/core.py
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/src/" src/parser.py
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/src/" src/printer.py
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/src/" src/time.py
+  install -Dm 755 -t "$pkgdir/usr/share/timeverter/src/" src/util.py
+  ln -s "$pkgdir/usr/share/timeverter/timeverter.py" "$pkgdir/usr/bin/timeverter"
```
Comment (noraj) [inline packages/timeverter/PKGBUILD:32]: `from src.core import main` won't work with this deployment. So you'll probably need a jointscript. cf. https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-generic-jointscript

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/timeverter/PKGBUILD (hunk 1 above)
Comment (D3vil0p3r) [reply packages/timeverter/PKGBUILD]: Thank you for the answer @noraj I updated it on the commit cbe35d1c40166ba84b3f4bd8c1299f825a77db9a

**Comment by noptrix** [comment]
@D3vil0p3r thanks for this - i am gonna do some updates on this and please refer to the new PKGBUILD file for future PRs / use our PKGBUILD templates.

**Comment by D3vil0p3r** [comment]
> @D3vil0p3r thanks for this - i am gonna do some updates on this and please refer to the new PKGBUILD file for future PRs / use our PKGBUILD templates.

Thank you @noptrix I will refer to the new PKGBUILD as indicated.

**Pull request merged**